airflow/sensors/external_task.py

```diff
@@ -121,7 +123,7 @@ class ExternalTaskSensor(BaseSensorOperator):
         or DAG does not exist (default value: False).
     """
 
-    template_fields = ["external_dag_id", "external_task_id", "external_task_ids"]
+    template_fields = ["external_dag_id", "external_task_id", "external_task_ids", "external_task_group_id"]
```

Comment:
```suggestion
    template_fields = ("external_dag_id", "external_task_id", "external_task_ids", "external_task_group_id")
```

Reply:
List works here as well, (Personally I prefer lists for a list of things.)

Reply:
hmm, I prefer to use a tuple if an update does not require.  just curious why you prefer a list here particularly.

Reply:
I think in most of the code base we have it as `template_fields: Sequence[str]`